assets/helpers/payPalContributionsCheckout/payPalContributionsCheckout.js

```diff
@@ -92,5 +92,5 @@ export function paypalContributionsRedirect(
       throw response;
     })
     .then((res) => { window.location = res.approvalUrl; })
-    .catch(() => errorHandler('Sorry, an error occurred, please try again or use another payment method.'));
+    .catch(errorHandler.bind(null, 'Sorry, an error occurred, please try again or use another payment method.'));
```

Comment:
Just for curiosity, what is the `.bind(null, ` for?

Reply:
It's just an alternative to an arrow function - `bind` creates a new function, the first parameter is what the value of `this` will be set to there. If this isn't conventional for the project I can change it back to the arrow